udata/tests/api/test_reuses_api.py

```diff
@@ -5,9 +5,11 @@
 
 from udata.core.badges.factories import badge_factory
 from udata.core.dataset.factories import DatasetFactory
+from udata.core.organization import metrics
```

Comment:
Are those imports needed? There were not there previously, and there's no other code added to the file, so this looks suspicious.

This shows that we don't have any CI action to catch those, maybe we'd want that?

Reply:
Good catch! Done in 924226d13